### ISSUE TITLE
Import data from Firefox (Microsoft Store)

### ISSUE THREAD

**Issue by manfromarce** [brave/brave-browser#32144]
## Description 

On Windows 10/11, Brave can't import bookmarks and data from from Microsoft Store version of Firefox. 
This occurs because profiles are located in `C:\Users\username\AppData\Local\Packages\Mozilla.Firefox_n80bbvh6b1yt2\LocalCache\Local\Mozilla\Firefox\Profiles` and 
`C:\Users\username\AppData\Local\Packages\Mozilla.Firefox_n80bbvh6b1yt2\LocalCache\Roaming\Mozilla\Firefox\Profiles` when Firefox uses MSIX packaging, not in `C:\Users\username\AppData\[Local|Roaming]\Mozilla\Firefox\Profiles`.

## Steps to Reproduce

Go to Settings > Get started > Import bookmarks and settings.

## Actual result:

Profiles from Firefox (MS Store version) are not listed.

## Expected result:

All Firefox profiles should be detected.

## Brave version (brave://version info)

1.56.20 Chromium: 115.0.5790.171 (Official Build) (64-bit) 

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Not tested
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

Chrome and other Chromium-based browsers have the same issue.

**Comment by gz83** [comment]
The user data of Firefox browser adopts a different storage format from that of chrome/chromium. If we want to import user data of Firefox browser, we need to introduce a third-party decryption library and implement specific details in the code

@manfromarce

**Comment by manfromarce** [comment]
Ok but importing from regular Firefox installations seems to be supported:
![image](https://github.com/brave/brave-browser/assets/45036600/76e0116b-8bc5-442f-ab53-3558dd5c87b1)

**Comment by gz83** [comment]
Yes, but it would be better if brave could support user data import for Firefox (Microsoft Store) version

@manfromarce
